configs/smartling-config.json

```diff
@@ -0,0 +1,102 @@
+{
+    "locales": [{
+            "smartling": "ar",
+            "application": "ar"
+        },
+        {
+            "smartling": "zh-CN",
+            "application": "zh-CN"
+        },
+        {
+            "smartling": "zh-TW",
+            "application": "zh-TW"
+        },
+        {
+            "smartling": "nl-NL",
+            "application": "nl"
+        },
+        {
+            "smartling": "fr-CA",
+            "application": "fr-CA"
+        },
+        {
+            "smartling": "fr-FR",
+            "application": "fr"
+        },
+        {
+            "smartling": "de-DE",
+            "application": "de"
+        },
+        {
+            "smartling": "hi-IN",
+            "application": "hi-IN"
+        },
+        {
+            "smartling": "id-ID",
+            "application": "id"
+        },
+        {
+            "smartling": "it-IT",
+            "application": "it"
+        },
+        {
+            "smartling": "ja-JP",
+            "application": "ja"
+        },
+        {
+            "smartling": "ko-KR",
+            "application": "ko"
+        },
+        {
+            "smartling": "ms-MS",
+            "application": "ms"
+        },
+        {
+            "smartling": "pl-PL",
+            "application": "pl"
+        },
+        {
+            "smartling": "pt-BR",
+            "application": "pt-BR"
+        },
+        {
+            "smartling": "pt-PT",
+            "application": "pt-PT"
+        },
+        {
+            "smartling": "ru-RU",
+            "application": "ru"
+        },
+        {
+            "smartling": "es",
+            "application": "es"
+        },
+        {
+            "smartling": "es-MX",
+            "application": "es-MX"
+        },
+        {
+            "smartling": "es-ES",
+            "application": "es-ES"
+        },
+        {
+            "smartling": "sw",
+            "application": "sw"
+        },
+        {
+            "smartling": "tr-TR",
+            "application": "tr"
+        },
+        {
+            "smartling": "vi-VN",
+            "application": "vi"
+        }
+    ],
+    "resourceSets": [{
+        "type": "FLUENT",
+        "authorizeContent": "false",
+        "pathRegex": "www-l10n\\/en\\/(?<path>.*)\\.ftl",
```

Comment:
I'm concerned about this pathRegex because of VPN stuff. That can be resolved a couple of ways though:
1) If VPN pages are manually pushed to prod, we can keep this path, initiate translations for these other languages, and they'll sit in the repo until pmac or someone pushes them to prod.
2) Update the path to grab everything *but* the products/VPN directory. This will tell Smartling to not import VPN stuff.

Reply:
This is how the VPN set of files are set up in Pontoon configuration: https://github.com/mozilla-l10n/www-l10n/blob/670f0f4d97899be6e772cb445cf269e7168698e1/configs/pontoon.toml#L253 - as you can see, it is for all locales. 

Exactly when and what is released on production is controlled on bedrock side.